src/styles/styles.js

```diff
@@ -59,18 +64,29 @@ export const styles = {
           sm: "1.7rem",
           xs: "1.045rem"
         }
-      }
+      },
+      helperText: "0.75rem"
     },
     globalDimensions: {
       percentageFull: "100%",
-      percentageThreeQuarter: "75%",
+      percentageNineTenths: "90%",
+      percentageThreeQuarters: "75%",
       percentageHalf: "50%",
+      percentageThird: "33%",
       percentageQuarter: "25%"
     },
     sidebarDimensions: {
       width: "200px",
       minWidth: "180px",
       maxWidth: "250px"
+    },
+    contactMeDimensions: {
```

Comment:
The idea is to make everything more generic so let's not specify per section if not for the whole application. The idea is you use the same spacing or widths in all the application. I would suggest move this to globaldimenssions and use relative values if necessary to avoid creating metrics for every section since we don't have things that different to specify something for a section.

Reply:
Please also start using Flex as a display which I believe his props and flow will help you out for this and the rest of the sections: https://css-tricks.com/snippets/css/a-guide-to-flexbox/